app/upgrades.go

```diff
@@ -117,6 +124,33 @@ func upgradeHandler(
 			"UpgradeTimeSetStakingRewardsPerSecond", communityParams.UpgradeTimeSetStakingRewardsPerSecond,
 		)
 
+		//
+		// Kavadist gov grant
+		//
+		msgGrant, err := authz.NewMsgGrant(
+			app.accountKeeper.GetModuleAddress(kavadisttypes.ModuleName),        // granter
+			app.accountKeeper.GetModuleAddress(govtypes.ModuleName),             // grantee
+			authz.NewGenericAuthorization(sdk.MsgTypeURL(&banktypes.MsgSend{})), // authorization
+			nil, // expiration
+		)
+		if err != nil {
+			return toVM, err
+		}
+		_, err = app.authzKeeper.Grant(ctx, msgGrant)
+		if err != nil {
+			return toVM, err
+		}
+		app.Logger().Info("created gov grant for kavadist funds")
+
+		//
+		// Gov Quorum
+		//
+		govTallyParams := app.govKeeper.GetTallyParams(ctx)
+		oldQuorum := govTallyParams.Quorum
+		govTallyParams.Quorum = sdkmath.LegacyMustNewDecFromStr("0.2").String()
```

Comment:
is this value ok for both mainnet & testnet?

Reply:
Ya think it's good to keep all env's the same